### DISCUSSION TITLE
Apache Web Server with Let's Encrypt - I need to do SSL Pass through

### DISCUSSION THREAD

**Discussion by hamdanfadi** [NginxProxyManager/nginx-proxy-manager#1473] [Q&A] [answered]
I'm using NPM since long time. So happy about. I came across an issue that I have a magento 2.4 server has Let's Encrypt on it. I need to do SSL Pass thought. I tried many ways. I couldn't manage to do it. Any Idea what I should do?

**Comment by chaptergy** [comment]
You could use a stream host, as suggested on [StackOverflow](https://stackoverflow.com/questions/38371840/ssl-pass-through-in-nginx-reverse-proxy)

**Comment by hamdanfadi** [comment]
Hi there,
This is not clear!
I have Apache web server as backend, and it has Let's Encrypt on it listen to 443 as you know not possible to change 443 to something else!.  Do I have to install Nginx in between Apache and NPM?

Cheers

**Reply by chaptergy** [reply]
Then it seems I don't understand what you mean. Don't you want to have ssl handled by your web server and the proxy just forwards that ssl encrypted packet?

**Comment by hamdanfadi** [comment]
Yes this is what I want exactly

**Comment by chaptergy** [comment, marked as answer]
You will need to have some patience, as SSL passthrough [might come in the next release](https://github.com/jc21/nginx-proxy-manager/pull/1479)
